samples/java/src/main/java/com/modulr/hmac/Hmac.java

```diff
@@ -0,0 +1,14 @@
+package com.modulr.hmac;
+
+import com.modulr.api.ModulrApiAuth;
+
+import java.util.Map;
+
+public class Hmac {
```

Comment:
This class is not referenced in the readme at all.

Reply:
Removing altogether since we point the reader to the sample project we provide.